app/src/main/res/navigation/main_nav_graph.xml

```diff
@@ -955,4 +963,6 @@
 
     <include app:graph="@navigation/cloud_backup_settings_graph" />
 
+    <include app:graph="@navigation/manual_backup_select_wallet_graph" />
```

Comment:
Do we need a separate graph for selecting a wallet? Maybe better to call it "manual_backup_graph"?